### PR TITLE
unified pipeline output

### PR DESCRIPTION
**Pull request by ruowan**
bump beta version

add trimSwaggerPath

bump version

append to log file

one json log one line.

fix env

github style path

upgrade oad to 0.7.1

collect error log

update package, small update about format

### CODE REVIEW

File: src/breaking-change.ts [hunk 1]
```diff
@@ -54,20 +92,73 @@ function blobHref(file: unknown) {
...
+      time: new Date(),
+      extra: {
+        mode: it.mode,
+      },
+      paths: [
+        {
+          tag: "New",
+          path: blobHref(
+            utils.trimGithubStyleFilePath(
+              utils.trimSwaggerPath(it.new.location || "")
```
Comment (jianyexi) [inline src/breaking-change.ts:142]: need to check if it.new is null

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/rest-api-specs-scripts",
-  "version": "0.9.1",
+  "version": "0.9.2",
```
Comment (jianyexi) [inline package.json:3]: to 0.10.0

File: src/utils.ts [hunk 3]
```diff
@@ -556,34 +618,51 @@ export const getOpenapiType = async function(configFile: string):Promise<string>
...
-     classic : classicLintVersion,
-     present :lintVersion
-  }
-}
+    classic: classicLintVersion,
+    present: lintVersion,
+  };
+};
+
+export const trimSwaggerPath = (filePath: string): string => {
```
Comment (jianyexi) [inline src/utils.ts:649]: pls change the name ,like getRelativeSwaggerPathToRepo

File: src/breaking-change.ts (hunk 1 above)
Comment (ruowan) [reply src/breaking-change.ts:142]: If oad runs no error, it will make sure to return `new` and `old` property.

File: package.json (hunk 2 above)
Comment (ruowan) [reply package.json:3]: good idea!

File: src/breaking-change.ts [hunk 4]
```diff
@@ -54,122 +91,138 @@ function blobHref(file: unknown) {
  * @param newSpec Path to the new swagger specification file.
  */
 async function runOad(oldSpec: string, newSpec: string) {
-  if (oldSpec === null || oldSpec === undefined || typeof oldSpec.valueOf() !== 'string' || !oldSpec.trim().length) {
-    throw new Error('oldSpec is a required parameter of type "string" and it cannot be an empty string.');
+  if (
+    oldSpec === null ||
+    oldSpec === undefined ||
+    typeof oldSpec.valueOf() !== "string" ||
+    !oldSpec.trim().length
```
Comment (zhenglaizhang) [inline src/breaking-change.ts:98]: actually this is check could extract as a common function

Comment (ruowan) [reply src/breaking-change.ts:98]: It's just format change. I will refactor code in the future.

**Pull request merged**